src/main/webapp/babel-src/util.js

```diff
@@ -6,10 +6,14 @@
  */
 import React from "react";
 
+// Regular old constants
 export const SEASON_NAMES_PRETTY = ["Fall", "Winter", "Summer"];
 export const SEASON_NAMES = SEASON_NAMES_PRETTY.map((season) => season.toLowerCase());
 export const DEFAULT_PROGRAM = "SOEN-General-Coop";
 export const EXPORT_TYPES = ["PDF", "MD", "TXT"];
+export const AUTO_SCROLL_PAGE_PORTION = 0.1; // auto scroll on the top and bottom 10% of screen
+export const AUTO_SCROLL_STEP = 10;
```

Comment:
does `AUTO_SCROLL_STEP` control how fast the screen moves up/down while dragging a course past the bottom or top? While testing I thought it might be a good idea to increase the speed of scrolling by a bit.

Reply:
What speed would you propose?

Reply:
Oh and yes lol